en/manual/audio/non-spatialized-audio.md

```diff
@@ -27,14 +27,14 @@ To play non-spatialized audio at runtime, create an instance of it and define it
 
 The [SoundInstance](xref:Stride.Audio.SoundInstance) controls audio at runtime with the following properties:
 
-| Property  | Function |
-|-------    |-------|
-| [IsLooping](xref:Stride.Audio.SoundInstance.IsLooping) | Gets or sets looping of the audio. |
-| [Pan](xref:Stride.Audio.SoundInstance.Pan)       | Sets the balance between left and right speakers. By default, each speaker a value of 0.5. |
-| [Pitch](xref:Stride.Audio.SoundInstance.Pitch)     | Gets or sets the audio pitch (frequency). |
-| [PlayState](xref:Stride.Audio.SoundInstance.PlayState)	| Gets the state of the [SoundInstance](xref:Stride.Audio.SoundInstance). |
-| [Position](xref:Stride.Audio.SoundInstance.Position)	| Gets the current play position of the audio. |
-| [Volume](xref:Stride.Audio.SoundInstance.Volume)	| Sets the audio volume. |
+| Property                                               | Function                                                                                 |
+|--------------------------------------------------------|------------------------------------------------------------------------------------------|
+| [IsLooping](xref:Stride.Audio.SoundInstance.IsLooping) | Gets or sets looping of the audio.                                                       |
+| [Pan](xref:Stride.Audio.SoundInstance.Pan)             | Sets the balance between left and right speakers. By default, each speaker a value of 0. |
```

Comment:
Looks like your change for the sentence `each speaker a value of` wasn't included

Reply:
Ohh

Reply:
Done